app/views/users/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="container mb-auto py-3 text-center">
+    <h1><%= current_user.email %></h1>
+    <h1><%= "Admin?: "+ current_user.admin.to_s %></h1>
```

Comment:
displaying true/false is really bad taste unless it is purely data output.
Maybe change this to display `You are an admin` or something

Reply:
Yes, this was a pure data output, just for me checking to validate what I was seeing. 
![image](https://user-images.githubusercontent.com/54109441/85349235-f29da580-b4cb-11ea-9045-3f091eb10737.png)

Did some quick work to make this a little more polished. Showing admin through a bad next to the email.